src/cell.scss

```
@@ -0,0 +1,17 @@
+$size: 100px;
+
+$shift: 103px;
+
+$start-days:
+  "mon" 0,
+  "tue" (1 * $shift),
+  "wed" (2 * $shift),
+  "thu" (3 * $shift),
+  "fri" (4 * $shift), 
+  "sat" (5 * $shift),
+  "sun" (6 * $shift);
```

```suggestion
$start-days:
  "mon" 1,
  "tue" 2,
  "wed" 3,
  "thu" 4,
  "fri" 5, 
  "sat" 6,
  "sun" 7;
```
And calculate values in the loop